diff -urN clamav-realtime-0.1.0.orig/files/clamav-realtime.desktop clamav-realtime-0.1.0/files/clamav-realtime.desktop
--- clamav-realtime-0.1.0.orig/files/clamav-realtime.desktop	2018-05-19 23:19:27.063315715 +0200
+++ clamav-realtime-0.1.0/files/clamav-realtime.desktop	2018-05-19 23:25:08.121245832 +0200
@@ -5,6 +5,7 @@
 Icon=clamav-realtime
 Terminal=false
 Type=Application
+Categories=System;
 MimeType=application/x-executable;
 X-GNOME-Autostart-enabled=true
 
