--- clamav-realtime-0.1.0.orig/main.c.orig	2016-05-26 21:48:41.028347455 +0200
+++ clamav-realtime-0.1.0/main.c	2016-05-26 21:49:10.468444379 +0200
@@ -89,7 +89,7 @@
 /* Create quarantine */
 struct stat file_stat;
 char path_quarantine[PATH_MAX];
-strcpy(path_quarantine, "/tmp");
+strcpy(path_quarantine, targetdir);
 strcat(path_quarantine, "/.quarantine/");
 
 if ( stat(path_quarantine, &file_stat) !=0) {
