diff -urN clamav-realtime-0.1.0.orig/Makefile clamav-realtime-0.1.0/Makefile
--- clamav-realtime-0.1.0.orig/Makefile	2018-10-23 23:52:17.813826936 +0200
+++ clamav-realtime-0.1.0/Makefile	2018-10-23 23:54:45.315388335 +0200
@@ -32,10 +32,10 @@
 		@cp files/$(NAME).png $(DESTDIR)/$(PIXMAPS)
 		install -d $(DESTDIR)/$(APPS)
 		@cp files/$(NAME).desktop $(DESTDIR)/$(APPS)
-		install -d $(DESTDIR)/$(DOCDIR)/$(NAME)-$(VERSION)
+		install -d $(DESTDIR)/$(DOCDIR)/$(PF)
 		for doc in $(DOCFILES); do \
-			cp $$doc $(DESTDIR)/$(DOCDIR)/$(NAME)-$(VERSION); \
-			bzip2 -9 $(DESTDIR)/$(DOCDIR)/$(NAME)-$(VERSION)/$$doc; \
+			cp $$doc $(DESTDIR)/$(DOCDIR)/$(PF); \
+			bzip2 -9 $(DESTDIR)/$(DOCDIR)/$(PF)/$$doc; \
 		done
 
 clean:
